lib/ruby_lsp/server.rb

```diff
@@ -690,6 +690,7 @@ def shutdown
 
     sig { params(config_hash: T::Hash[String, T.untyped]).void }
     def perform_initial_indexing(config_hash)
+      RubyIndexer::IndexRbs.new(@global_state.index).index_core_classes
```

Comment:
Currently this is fast, so I don't think need to have any logging or progress indicator. Maybe we will once it's expanded to methods, etc.

Reply:
How fast is it on a Core Spin instance?